RIPS/rip-7560.md

```diff
@@ -61,7 +61,7 @@ smart contracts. This, however, is impossible without an addition of Native Acco
 | Name                  | Value                                                                           |
 |-----------------------|---------------------------------------------------------------------------------|
 | FORK_BLOCK            | 	TBD                                                                            |
-| AA_TX_TYPE            | 4                                                                               |
+| AA_TX_TYPE            | 126                                                                             |
```

Comment:
I''m afraid that 126 is a deposit tx type at Optimism. Relative docs at optimism specs [here](https://specs.optimism.io/protocol/deposits.html#the-deposited-transaction-type).